homeassistant/components/climate/melissa.py

```diff
@@ -26,7 +26,7 @@
                  SUPPORT_ON_OFF | SUPPORT_TARGET_TEMPERATURE)
 
 OP_MODES = [
-    STATE_AUTO, STATE_COOL, STATE_DRY, STATE_FAN_ONLY, STATE_HEAT
```

Comment:
Why did you remove this? Does melissa not support any auto mode? If so, you should also remove it from the translation functions at the bottom of this file. Also, that makes this PR a breaking change - if someone uses 'auto' in e.g. his automations, this will throw an exception once this PR is merged.

Reply:
Hi! Yes there must have been an update in their api and this option is now gone (afaik)

Reply:
It is also unlikely anyone is using it, when I try to use that mode the underlaying api throws an exception